### DISCUSSION TITLE
EQ2 MyOpenMath

### DISCUSSION THREAD

**Discussion by AbbyANoble** [TeamBasedInquiryLearning/precalculus#327] [General]
Reviewer: @tdegeorge 

The homework set is far too long right now, but I thought it may be good to leave different types we can choose from. Some require that you put in a linear equation and the answer, some just the answer. Let me know!

Right now, they are sorted in the following "categories":

Questions 1-8: Mixtures, continuous (chemicals, lbs of coffee/candy/etc)
Questions 9-13: Mixtures, financial with interest rates
Questions 14-19: Mixtures, discrete (coins, types of bills, tickets)
Questions 20-31: Distance, rate, time
Questions 32-36: Geometry related (which isn't in the activity book, but I through these in. Should we add one to the activity book like these?)

**Comment by tdegeorge** [comment]
I do think the homework is long, but I like that there are multiple ones to choose from. I would suggest leaving them so people can pick and choose which ones they want.

I also like the geometry ones - especially the perimeter/area questions. It probably would be good to have a similar question in the activity book!

**Reply by AbbyANoble** [reply]
I'll make an issue that we should (eventually) write a geometry one.
